### DISCUSSION TITLE
Neuralcoref behind firewall

### DISCUSSION THREAD

**Discussion by morankyle** [explosion/spaCy#8461] [Help: Coding & Implementations]
Is it possible to utilize neuralcoref from behind a firewall? After using conda install and importing neuralcoref with Spacy v2 I get this error message. 

HTTPSConnectionPool...... Max retires exceeded with url: /models/huggingface/neuralcoref.tar.gz)

**Comment by polm** [comment]
neuralcoref requires a model, which is downloaded the first time you import it. After that an Internet connection is not required and it doesn't contact the Internet in any way. 

Are you sure your firewall is the issue? This is the model URL:

https://s3.amazonaws.com/models.huggingface.co/neuralcoref/neuralcoref.tar.gz
